src/contract.ts

```diff
@@ -0,0 +1,31 @@
+import assert from "assert";
+import fs from "fs";
+import Web3 from "web3";
+import type { AbiItem } from "web3-utils";
+import { isAddress } from "web3-utils";
+import type { Contract } from "web3-eth-contract";
+import path from "path";
+
+declare const CONFIG: ConfigType;
+
+export class AllowedListContract {
+    web3: Web3;
+    allowedListContract: Contract;
+
+    constructor() {
+        this.web3 = new Web3(CONFIG.contract.nodeUrl);
+
+        // todo move file directories to config 
+        const accountRules = JSON.parse(fs.readFileSync(path.resolve(__dirname, "./contract/AccountRules.json"), "utf-8"));
+        const accountRulesList = JSON.parse(fs.readFileSync(path.resolve(__dirname, "./contract/AccountRulesList.json"), "utf-8"));
+        const chainId = CONFIG.contract.chainId;
+
+        assert(isAddress(accountRules["networks"][chainId]["address"]));
+        this.allowedListContract = new this.web3.eth.Contract(accountRulesList["abi"] as AbiItem[], accountRules["networks"][chainId]["address"]);
```

Comment:
Hardcoded account rules contract may become obsolete. Please fetch the correct current address of account rules contract from the ingress contract as shown here: https://github.com/ConsenSys/permissioning-smart-contracts/blob/main/src/chain/contracts/AccountRules.ts

Reply:
I would love if [the project](https://github.com/ConsenSys/permissioning-smart-contracts/tree/main/src/chain) -- i.e. including the permissioning dapp -- had a npm package, but sadly it doesn't. So let's just mirror how they do it. There should even be functionality to show the full allowlist somewhere in the app we should copy over.

Reply:
Yeah, I agree, this will be much better. Although wasn't sure if we want to copy stuff, but there's no other way. Thanks